docs/Installing.asciidoc

```diff
@@ -279,7 +279,7 @@ ensure shared storage between openQA WebUI and workers.
 Directory +/var/lib/openqa/share+ contains all required data and should be
 shared with read-write access across all nodes present in openQA cluster.
 This step is intentionally left on system administrator to choose proper shared
-storage for her specific needs.
```

Comment:
;-) My proposal is to just delete the whole sentence as it is IMHO not adding any relevant information. The *should be* in the previous sentence is ambiguous enough ;-)